tools/build.sbt

```diff
@@ -220,3 +229,55 @@ lazy val cloudflowOperator =
       dockerUsername := sys.props.get("docker.username"),
       dockerRepository := sys.props.get("docker.registry"),
       dockerBaseImage := "adoptopenjdk/openjdk11:alpine-jre")
+
+lazy val cloudflowDescriptorGenerator =
+  Project(id = "cloudflow-descriptor-generator", base = file("cloudflow-descriptor-generator"))
```

Comment:
maybe `cloudflow-descriptor` is good enough as name (also to reflect as cloudflow.descriptor as package)